mysql/datadog_checks/mysql/mysql.py

```diff
@@ -851,7 +866,39 @@ def _query_size_per_schema(self, db):
 
                 return schema_size
         except (pymysql.err.InternalError, pymysql.err.OperationalError) as e:
-            self.warning("Avg exec time performance metrics unavailable at this time: %s", e)
+            self.warning("Size of schemas metrics unavailable at this time: %s", e)
+
+        return {}
+
+    def _query_size_per_table(self, db, extra=False):
+        try:
+            with closing(db.cursor()) as cursor:
+                if extra:
+                    cursor.execute(SQL_QUERY_EXTRA_TABLE_SIZE)
+                else:
+                    cursor.execute(SQL_QUERY_TABLE_SIZE)
+
+                if cursor.rowcount < 1:
+                    self.warning("Failed to fetch records from the information schema 'tables' table.")
+                    return None
+
+                table_index_size = {}
+                table_data_size = {}
+                for row in cursor.fetchall():
+                    table_schema = str(row[0])
+                    table_name = str(row[1])
+                    index_size = long(row[2])
+                    data_size = long(row[3])
+
+                    # set the tag as the dictionary key
+                    table_index_size["schema:{0}".format(table_schema)] = index_size
+                    table_index_size["table:{0}".format(table_name)] = index_size
+                    table_data_size["schema:{0}".format(table_schema)] = data_size
+                    table_data_size["table:{0}".format(table_name)] = data_size
```

Comment:
so, I read the code wrong originally and it looks like this **won't work to tag these metrics properly with schema and table**

In order to make it work, I added a hack [right here](https://github.com/DataDog/integrations-core/commit/a7c703546f993f4339f4636de1d877476a0a84cd). This isn't great. This does give me the correct results though, when I run the agent locally. There might be a better approach by mucking around with this method https://github.com/DataDog/integrations-core/blob/master/mysql/datadog_checks/mysql/collection_utils.py#L11 to allow for returning multiple tags for a single, unique value. Not sure if that's worth it. 

@alexandre-normand, sorry to bring you in here, but do you have any thoughts on this?

Reply:
> In order to make it work, I added a hack right here.

This seems fine to me. It might be nice to refactor this code to define better types that held the additional tags as a list but this would be a big effort that shouldn't block this PR.